app/controllers/coronavirus_form/offer_space_controller.rb

```diff
@@ -22,6 +22,8 @@ def submit
       render "coronavirus_form/#{PAGE}"
     elsif session["check_answers_seen"]
       redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    elsif offer_space === "Yes"
```

Comment:
I'm a little unsure if `===` or `==` is better pratice here.
I always get confused coming back from JS.

I know that `===` is implimented differently depending on the parent class.
But this seems to work here with the string matching, so feel free to ignore if this is pedentry!

Reply:
Would `.eql?` be more Ruby?

Reply:
I've changed it to `.eql?` :)